### PR TITLE
Update projects using Verdaccio and their stars

### PR DESCRIPTION
**Pull request by bl-ue**
* rearrange some based on the star count
* remove two duplicates
* add star count to "workshopper how to npm" because they
   recently achieved 1k stars

### CODE REVIEW

**Review by juanpicado — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -211,25 +211,22 @@ You can find and chat with then over Discord, click [here](http://chat.verdaccio
...
-* [ethereum/web3.js](https://github.com/ethereum/web3.js) *(+8k ⭐️)*
+* [create-react-app](https://github.com/facebook/create-react-app/blob/master/CONTRIBUTING.md#contributing-to-e2e-end-to-end-tests) *(+86.2k ⭐️)*
+* [Gatsby](https://github.com/gatsbyjs/gatsby) *(+49.2k ⭐️)*
+* [Babel.js](https://github.com/babel/babel) *(+38.5k ⭐️)*
+* [Vue CLI](https://github.com/vuejs/vue-cli) *(+27.4k ⭐️)*
+* [Angular CLI](https://github.com/angular/angular-cli) *(+24.3k ⭐️)*
+* [Uppy](https://github.com/transloadit/uppy) *(+23.8k ⭐️)*
+* [bit](https://github.com/teambit/bit) *(+13k ⭐️)*
+* [Aurelia Framework](https://github.com/aurelia/framework) *(+11.6k ⭐️)*
 * [pnpm](https://github.com/pnpm/pnpm) *(+9.1k ⭐️)*
```
Comment (bl-ue) [inline README.md:222]: Oops! forgot this.
```suggestion
* [pnpm](https://github.com/pnpm/pnpm) *(+10.1k ⭐️)*
```

File: README.md [hunk 2]
```diff
@@ -211,25 +211,22 @@ You can find and chat with then over Discord, click [here](http://chat.verdaccio
...
-* [bit](https://github.com/teambit/bit) *(+12k ⭐️)*
-* [ethereum/web3.js](https://github.com/ethereum/web3.js) *(+8k ⭐️)*
-* [pnpm](https://github.com/pnpm/pnpm) *(+9.1k ⭐️)*
-* [ethereum/web3.js](https://github.com/ethereum/web3.js) *(+9.4k ⭐️)*
-* [NX](https://github.com/nrwl/nx) *(+5.2k ⭐️)*
-* [webiny-js](https://github.com/webiny/webiny-js) *(+3.7k ⭐️)*
-* [Mozilla Neutrino](https://github.com/neutrinojs/neutrino) *(+3k ⭐️)*
-* [Hyperledger Composer](https://github.com/hyperledger/composer) *(+1.6k ⭐️)*
-* [webiny-js](https://github.com/Webiny/webiny-js) *(+1k ⭐️)*
+* [create-react-app](https://github.com/facebook/create-react-app/blob/master/CONTRIBUTING.md#contributing-to-e2e-end-to-end-tests) *(+86.2k ⭐️)*
```
Comment (bl-ue) [inline README.md:214]:
```suggestion
* [create-react-app](https://github.com/facebook/create-react-app/blob/master/CONTRIBUTING.md#customizing-e2e-registry-configuration) *(+86.2k ⭐️)*
```

**Review by juanpicado — APPROVED** [review]
LGTM

**Comment by juanpicado** [comment]
thanks @bl-ue highly appreciated !

**Pull request merged**